packages/ui-extensions/src/surfaces/checkout/components/Icon/examples/basic-icon.example.html

```diff
@@ -1,5 +1,4 @@
-<s-icon type="arrow-left" />
-<s-icon type="arrow-right" />
-<s-icon type="cart" />
-<s-icon type="x" />
-<s-icon type="check-circle" />
+<s-icon type="store" />
+<s-icon type="settings" />
+<s-icon type="star" />
```

Comment:
```suggestion
<s-icon type="star" />
<s-icon type="settings" />
```

to match the image order.